Classes/Popover.swift

```diff
@@ -117,14 +118,23 @@ open class Popover: UIView {
 
   open func show(_ contentView: UIView, fromView: UIView, inView: UIView) {
     let point: CGPoint
+    
+    if self.popoverType == .auto {
+        if let point = fromView.superview?.convert(fromView.frame.origin, to: nil), point.y + contentView.frame.height + 75 > inView.frame.height {
```

Comment:
Thanks for a nice PR!
I am concerned about the magic number of 75, how have you decided this number?

Reply:
I agree the number is too magic there. Update this part, so that it takes care about fromView height (it might be bigger than expected) as well as arrow height (it can have any height, because it is customizable). The decision between .up and .down should be much more accurate now, without magic numbers.
Thanks for the heads up!